eth/client.go

```diff
@@ -602,13 +417,7 @@ func (c *client) Vote(pollAddr ethcommon.Address, choiceID *big.Int) (*types.Tra
 		return nil, err
 	}
 
-	gl, gp := c.GetGasInfo()
-	opts, err := c.accountManager.CreateTransactOpts(gl, gp)
-	if err != nil {
-		return nil, err
-	}
-
-	return poll.Vote(opts, choiceID)
+	return poll.Vote(c.defaultTxOpts, choiceID)
```

Comment:
What's the benefit of introducing defaultTxOpts instead of the existing code which uses the value returned by CreateTransactOpts in the Vote() method? Seems like that would've worked fine.